include/drivers/kscan.h

```diff
@@ -24,6 +24,12 @@
 extern "C" {
 #endif
 
+/* Drive all columns of keyboard matrix to low */
+#define KEYBOARD_COLUMN_DRIVE_ALL -2
+/* Tri-state all columns of keyboard matrix */
+#define KEYBOARD_COLUMN_DRIVE_NONE -1
```

Comment:
please use Doxygen comments, `/**` and prefix macros with `KSCAN_`

Reply:
Done